isso/db/comments.py

```diff
@@ -81,6 +81,25 @@ def activate(self, id):
             '    mode=1',
             'WHERE id=? AND mode=2'], (id, ))
 
+    def is_previously_approved_author(self, email):
+        """
+        Search for previously activated comments with this author email.
+        """
+
+        if type(email) is str and len(email) >= 3:
```

Comment:
What is this line trying to guard against?

Reply:
in some setups, email is not required, in which case this function gets a None (see views/comments.py), hence the `str` check.

The `len(email) >= 3` is to guard against cases where an invalid email, or even a blank string was somehow passed.

Reply:
Can we just explicitly check for None and perhaps add a comment about the length? That will hopefully easier to understand for others who read this code later.